api/desecapi/models.py

```diff
@@ -232,7 +232,7 @@ class RenewalState(models.IntegerChoices):
     published = models.DateTimeField(null=True, blank=True)
     minimum_ttl = models.PositiveIntegerField(default=get_minimum_ttl_default)
     renewal_state = models.IntegerField(choices=RenewalState.choices, default=RenewalState.FRESH)
-    renewal_changed = models.DateTimeField(null=True, blank=True)
+    renewal_changed = models.DateTimeField(auto_now_add=True)
```

Comment:
maybe the migration needs updating, too?

Reply:
did that: https://github.com/desec-io/desec-stack/pull/405/commits/b9d6b0bc09e8e6c64f78fcd49be0f496c4f2d7ca#diff-e400995957e27d4fc5051dfb90123487R30

Reply:
sorry, overlooked. all good :sweat_smile: